core/src/trezor/ui/layouts/tt/recovery.py

```diff
@@ -10,6 +10,11 @@
 CONFIRMED = trezorui2.CONFIRMED  # global_import_cache
 INFO = trezorui2.INFO  # global_import_cache
 
+if TYPE_CHECKING:
+    from typing import List, Set, Tuple
```

Comment:
please use `list`, `set`, `tuple` etc ... the capitalized versions are compatible with older python versions which we don't need in firmware

Reply:
fixed
https://github.com/trezor/trezor-firmware/pull/4142/commits/3a03e8b2059ca96f7c3dacfae978ccd3ed24fb9f